docs/tutorials/clickstream-docker.rst

```diff
@@ -164,17 +166,11 @@ Load the Streaming Data to KSQL
 
 #.  Launch the KSQL CLI in Client Server Mode.
```

Comment:
This should just be "Launch the KSQL CLI".  The CLI only has no other mode today than client-server.